e2e/provision/README.md

```diff
@@ -167,23 +193,26 @@ then push the updates back to the Draft revision. You will use `kpt` and the
 To pull the package to a local directory, you use the `rpkg pull` command:
 
 ```bash
-kpt alpha rpkg pull -n default mgmt-08c26219f9879acdefed3469f8c3cf89d5db3868 regional
+kpt alpha rpkg pull -n default mgmt-08c26219f9879acdefed3469f8c3cf89d5db3868 /tmp/nephio-e2e-lab/regional
```

Comment:
curious as to why /tmp directory instead of $HOME/.kube?

Reply:
Well, this instruction creates the `regional` folder in the current directory, so providing an absolute path can help to make that distinction

Reply:
Same comment. Why do we need to change the path ?